frontend/src/components/editProject/EditCollaboratingOrganizations.js

```diff
@@ -0,0 +1,53 @@
+import { List } from "@material-ui/core";
+import React from "react";
+
+import MiniOrganizationPreview from "../organization/MiniOrganizationPreview";
+import AutoCompleteSearchBar from "../search/AutoCompleteSearchBar";
+
+export default function EditCollaboratingOrganizations({
+  texts,
+  baseUrl,
+  searchBarClass,
+  handleAddCollaboratingOrg,
+  organizationPreviewsContainerClass,
+  organizationPreviewClass,
+  selectedCollaboratingOrgs,
+  handleDeleteCollaboratingOrg,
+}) {
+  const renderSearchOption = (option) => {
+    return <React.Fragment>{option.name}</React.Fragment>;
+  };
+
+  return (
+    <>
+      {/* Currently allows any org to be added, maybe needs some verification on selected orgs part*/}
```

Comment:
Do you mind updating this comment to include `TODO:` in case we need to follow up on it?

Reply:
Sure thing 👍